### ISSUE TITLE
BadZipFile: File is not a zip file

### ISSUE THREAD

**Issue by Srikar-Ganta** [naveen521kk/manim-fonts#44]
When I am using the registr font, then i am getting the above error. 

My observation:
- when file is getting downloaded, it is getting courrputed. It tried to extract the downloaded file manually, then it is showing error. 
When I download from google fonts, manually and unzip it, it is unzipping properly. 

```
from manim import *
from manim_fonts import *


# dimensions of video
class QuadraticEquation(Scene):
    def construct(self):
        
        with RegisterFont("Poppins") as fonts:
            print(fonts)
            heading_text = Text("Let's solve the quadratic equation", font_size=30)
            heading_text.to_edge(UP, buff=0.5)
            self.play(Write(heading_text))
            self.wait(1)
```

**Comment by naveen521kk** [comment]
Looks like google-fonts changed their API to download fonts, I'll update the package soon to support the new API.

**Comment by naveen521kk** [comment]
Hey @Srikar-Ganta,

I've just released v0.4.0 - you could run `pip install -U manim-fonts` to update. Let me know if this update fixes this issue.

Thanks!
